libsql-client/examples/select.rs

```diff
@@ -70,7 +71,16 @@ async fn bump_counter(db: impl Connection) -> String {
 
 #[tokio::main]
 async fn main() {
-    let db = libsql_client::reqwest::Connection::connect_from_env().unwrap();
-    let response = bump_counter(db).await;
-    println!("{response}")
+    match libsql_client::reqwest::Connection::connect_from_env() {
+        Ok(remote_db) => {
+            let response = bump_counter(remote_db).await;
+            println!("Remote:\n{response}");
+        }
+        Err(e) => println!("Failed to fetch from a remote database: {e}"),
+    }
+
+    let local_db =
+        libsql_client::local::Connection::connect("/tmp/libsql_client_test_db.db").unwrap();
```

Comment:
Could you use this to get at the environment-approved tmp directory (which will definitely not be /tmp on some systems).

https://doc.rust-lang.org/std/env/fn.temp_dir.html